setup.py

```diff
@@ -59,9 +59,10 @@
     "Werkzeug<2.1",
     "flask-celeryext<0.5.0",
     "gitpython>=3.1",
-    "marshmallow>2.13.0,<=2.20.1",
+    "marshmallow>2.13.0,<3",
     "reana-commons[kubernetes,yadage,snakemake,cwl]>=0.95.0a2,<0.96.0",
-    "reana-db>=0.95.0a2,<0.96.0",
+    # commenting out so that I can test newer dependencies
+    # "reana-db>=0.95.0a2,<0.96.0",
```

Comment:
TODO: uncomment after https://github.com/reanahub/reana-db/pull/234 has been merged and reana-db has been released